src/components/ha-selector/ha-selector-action.ts

```diff
@@ -18,19 +18,22 @@ export class HaActionSelector extends LitElement {
 
   @property({ type: Boolean, reflect: true }) public disabled = false;
 
-  private _actions = memoizeOne((action: Action | undefined) => {
-    if (!action) {
-      return [];
+  // Add path here to ignore memoize if the path changes
+  private _actions = memoizeOne(
+    (action: Action | undefined, _path?: ItemPath) => {
```

Comment:
The only reason this works is because we create (the same) path on every render in the parent, this is not the proper fix

Reply:
I will open a revert and reopen the solution from @wendevlin since the only proper solution would probably to modify the trigger/action array reference.